### DISCUSSION TITLE
Scramble::routes alternative

### DISCUSSION THREAD

**Discussion by WildEgo** [dedoc/scramble#364] [Q&A]
Hey, finally had some free time to play around with the library once again, noticed that on the current `main` branch `Scramble::routes` is no longer called therefore I no longer have access to the `$routeResolver`, was it replaced or is there any new way of doing it?

**Comment by WildEgo** [comment]
I'll close this since I can fix my issue by force registering scramble on my own service provider, still weird behavior as `resolveTagsUsing` still works

**Reply by romalytvynenko** [reply]
Hey @WildEgo 
Not sure I understand what you've meant. Feel free to elaborate if you think it is relevant

**Reply by WildEgo** [reply]
Something had changed from the last time I had time to play with the package and I'm now forced to add `$this->app->register(ScrambleServiceProvider::class, true);`, before for some reason it just worked and the tags still work in that code even without registering.

https://github.com/EVOCATIVE-AE/filament-openapi/commit/9959b6ae0cc119bcce574975a88370fdf8c626ff

**Reply by romalytvynenko** [reply]
@WildEgo I'm not sure this is related to Scramble's code
